### ISSUE TITLE
`if` block with more than two conditions are not evaluated

### ISSUE THREAD

**Issue by vasuadari** [bettyblocks/liquid-elixir#88]
```Elixir
iex(6)> template_body = """
...(6)>   {% if v1 == 1 and v2 == 2 and v3 == 3 %} Hello {% endif %}
...(6)> """
"  {% if v1 == 1 and v2 == 2 and v3 == 3 %} Hello {% endif %}\n"
iex(7)>
nil
iex(8)> Liquid.Template.parse(template_body)
%Liquid.Template{
  blocks: [],
  presets: %{},
  root: %Liquid.Block{
    blank: false,
    condition: nil,
    elselist: [],
    iterator: [],
    markup: nil,
    name: :document,
    nodelist: [
      "  ",
      %Liquid.Block{
        blank: false,
        condition: %Liquid.Condition{
          child_condition: %Liquid.Condition{
            child_condition: nil,
            child_operator: nil,
            left: %Liquid.Variable{
              filters: [],
              literal: nil,
              name: "v3",
              parts: ["v3"]
            },
            operator: :==,
            right: %Liquid.Variable{
              filters: [],
              literal: 3,
              name: "3",
              parts: []
            }
          },
          child_operator: :and,
          left: %Liquid.Variable{
            filters: [],
            literal: nil,
            name: "v2",
            parts: ["v2"]
          },
          operator: :==,
          right: %Liquid.Variable{filters: [], literal: 2, name: "2", parts: []}
        },
        elselist: [],
        iterator: [],
        markup: "v1 == 1 and v2 == 2 and v3 == 3",
        name: :if,
        nodelist: [" Hello "],
        parts: []
      },
      "\n"
    ],
    parts: []
  }
}
```
**Expected output:**
```
iex(1)> template_body = """
...(1)>   {% if v1 == 1 and v2 == 2 and v3 == 3 %} Hello {% endif %}
...(1)> """
"  {% if v1 == 1 and v2 == 2 and v3 == 3 %} Hello {% endif %}\n"
iex(2)> Liquid.Template.parse(template_body)
%Liquid.Template{
  blocks: [],
  errors: [],
  presets: %{},
  root: %Liquid.Block{
    blank: false,
    condition: nil,
    elselist: [],
    iterator: [],
    markup: nil,
    name: :document,
    nodelist: [
      "  ",
      %Liquid.Block{
        blank: false,
        condition: %Liquid.Condition{
          child_condition: %Liquid.Condition{
            child_condition: %Liquid.Condition{
              child_condition: nil,
              child_operator: nil,
              left: %Liquid.Variable{
                filters: [],
                literal: nil,
                name: "v3",
                parts: ["v3"]
              },
              operator: :==,
              right: %Liquid.Variable{
                filters: [],
                literal: 3,
                name: "3",
                parts: []
              }
            },
            child_operator: :and,
            left: %Liquid.Variable{
              filters: [],
              literal: nil,
              name: "v2",
              parts: ["v2"]
            },
            operator: :==,
            right: %Liquid.Variable{
              filters: [],
              literal: 2,
              name: "2",
              parts: []
            }
          },
          child_operator: :and,
          left: %Liquid.Variable{
            filters: [],
            literal: nil,
            name: "v1",
            parts: ["v1"]
          },
          operator: :==,
          right: %Liquid.Variable{filters: [], literal: 1, name: "1", parts: []}
        },
        elselist: [],
        iterator: [],
        markup: "v1 == 1 and v2 == 2 and v3 == 3",
        name: :if,
        nodelist: [" Hello "],
        parts: [],
        strict: true
      },
      "\n"
    ],
    parts: [],
    strict: true
  }
}
```

**Comment by Spadavecchia** [comment]
Can you write a sample?

**Comment by vasuadari** [comment]
@Spadavecchia Added sample to the description.

**Comment by nulian** [comment]
Merged and released in 0.9.0
